CONTRIBUTING.rst

```diff
@@ -333,6 +332,32 @@ Recommendation for the Pull Requests:
    -  If applicable, ensure that your fix includes appropriate tests. Adding tests for your bug fix helps prevent future regressions and maintains the stability of the software.
 
 
+Contributing to test data
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your changes include updating test data, you may work with the ``gpm/tests/data`` directory as you would with any other git repository.
+You need to ask the maintainers to become a contributor on the `gpm_api_test_data <https://github.com/ghiggi/gpm_api_test_data>`_ repository to create a branch and a Pull Request.
+The API repository keeps track of the currently checked-out commit of the test-data repository. When the checked-out commit changes, you can register this change in the API repository by running
+
+.. code-block:: bash
+
+    git add gpm/tests/data
+
+and committing.
+
+
+To submit your contribution that involves modifying test data, please follow this procedure.
+
+(A: API repository, B: test-data repository)
+
+1. Make a *feature branch* for B
```

Comment:
Should we specify maybe that to make a feature branch the contributor should first go within the gpm/tests/data directory?